src/libcore/iter/sources.rs

```diff
@@ -209,7 +199,7 @@ unsafe impl<A, F: FnMut() -> A> TrustedLen for RepeatWith<F> {}
 /// assert_eq!(None, pow2.next());
 /// ```
 #[inline]
-#[unstable(feature = "iterator_repeat_with", issue = "48169")]
+#[stable(feature = "iterator_repeat_with", since = "1.28.0")]
 pub fn repeat_with<A, F: FnMut() -> A>(repeater: F) -> RepeatWith<F> {
```

Comment:
There is some documentation re. DEI left that needs to be removed:

> An iterator produced by `repeat_with()` is a `DoubleEndedIterator`. It is important to note that reversing `repeat_with(f)` will produce the exact same sequence as the non-reversed iterator. In other words, `repeat_with(f).rev().collect::<Vec<_>>()` is equivalent to `repeat_with(f).collect::<Vec<_>>()`.

I would replace this with a note saying that it isn't DEI and to ask users who need that to come forward and file an issue explaining their use case.

Reply:
Fixed.